app/delivery/send_to_providers.py

```diff
@@ -138,6 +126,14 @@ def send_sms_to_provider(notification):
     return message_id
 
 
+def _get_verify_code(notification):
+    key = f"2facode-{notification.id}".replace(" ", "")
+    recipient = redis_store.get(key)
+    if recipient:
+        recipient = recipient.decode("utf-8")
```

Comment:
Just wondering if this would be better served with EAFB strategies:

```
from contextlib import suppress

...

with suppress(AttributeError):
    recipient = redis_store.get(key).decode("utf-8")
```

very minor change, but eliminates an extra `if` statement, and is cleaner from a Python standpoint. Otherwise does the same thing.